README.md

```diff
@@ -1 +1,61 @@
-# julia-container-scripts
\ No newline at end of file
+# julia-container-scripts
```

Comment:
Maybe `julia-container-build-scripts` or `julia-docker-build-scripts` is a better name?